indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -47,6 +68,9 @@
 
   private static final Set<Path> existing = Sets.newHashSet();
 
+  private static final int NUM_RETRIES = 6;
```

Comment:
6 seems a bit excessive, is that consistent with what we use for our DataSegmentPushers?